dist/secure-handlebars.js

```diff
@@ -10658,15 +10697,100 @@ StrictContextParser.prototype.getCurrentState = function() {
 };
 
 
+// <iframe srcdoc=""> is a scriptable attribute too
+// Reference: https://html.spec.whatwg.org/multipage/embedded-content.html#attr-iframe-srcdoc
+var scriptableTags = {
+    script:1,style:1,
+    svg:1,xml:1,math:1,
+    applet:1,object:1,embed:1,link:1,
+    scriptlet:1                  // IE-specific
+};
 
+/**
+ * @function StrictContextParser#isScriptableTag
+ *
+ * @returns {boolean} true if the current tag can possibly incur script either through configuring its attribute name or inner HTML
+ *
+ * @description
+ * Check if the current tag can possibly incur script either through configuring its attribute name or inner HTML
+ *
+ */
+StrictContextParser.prototype.isScriptableTag = function() {
+    return scriptableTags[this.tagNames[0]] === 1;
+};
 
+// Reference: http://www.w3.org/TR/html-markup/elements.html
+StrictContextParser.ATTRIBUTE_NAME_URI_TYPE  = 1,
+StrictContextParser.ATTRIBUTE_NAME_CSS_TYPE  = 2,
+StrictContextParser.ATTRIBUTE_NAME_SCRIPTABLE_TYPE = 3,
+StrictContextParser.ATTRIBUTE_NAME_MIME_TYPE = 4,
+StrictContextParser.ATTRIBUTE_NAME_GENERAL_TYPE = undefined;
+
+var attributeNamesType = {
+    // we generally do not differentiate whether these attribtues are tag specific during matching for simplicity
+    'href'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for a, link, img, area, iframe, frame, video, object, embed ...
+    'src'        :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,
+    'background' :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for body, table, tbody, tr, td, th, etc? (obsolete)
+    'action'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for form, input, button
+    'formaction' :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     
+    'cite'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for blockquote, del, ins, q
+    'poster'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for img, object, video, source
+    'usemap'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for image
+    'longdesc'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,                         
+    'folder'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for a
+    'manifest'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for html
+    'classid'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for object
+    'codebase'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for object, applet
+    'data'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for object
+    'icon'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for command
+    'profile'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for head
+    'content'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for meta http-equiv=refresh, kill more than need
+
```

Comment:
can we make "data" and "content" tag specific? these are names that sound more common for people to put arbitrary text there, not necessarily URI.